main/main.cpp

```diff
@@ -262,25 +262,20 @@ void Main::print_help(const char *p_binary) {
 	OS::get_singleton()->print("  -h, --help                       Display this help message.\n");
 	OS::get_singleton()->print("  --version                        Display the version string.\n");
 	OS::get_singleton()->print("  -v, --verbose                    Use verbose stdout mode.\n");
-	OS::get_singleton()->print(
-			"  --quiet                          Quiet mode, silences stdout messages. Errors are still displayed.\n");
+	OS::get_singleton()->print("  --quiet                          Quiet mode, silences stdout messages. Errors are still displayed.\n");
```

Comment:
This is longer than 120 chars, which is why it was split by `black`. Why does CI pass?

Reply:
Because it's `clang-format`? 😮

I dunno why split though, quite ugly if you ask me, breaks alignment heavily.

Reply:
Welp nevermind me. I don't know why those had been split either.

Reply:
Looks like it was done by 579342810f5c453e91063c54ca6544300ea09090 who is known for struggling with respecting the style guide :P